src/app/organization/fund-mapping/fund-mapping.component.html

```
@@ -36,140 +36,125 @@
+          <mat-option value="approvalDate">{{ 'labels.inputs.Approval Date' | translate }}</mat-option>
+          <mat-option value="createdDate">{{ 'labels.inputs.Creation Date' | translate }}</mat-option>
+          <mat-option value="disbursalDate">{{ 'labels.catalogs.Disbursement Date' | translate }}</mat-option>
         </mat-select>
       </mat-form-field>
 
       <mat-form-field fxFlex="31%" (click)="fromDatePicker.open()">
-        <mat-label>From Date</mat-label>
-        <input matInput [min]="minDate" [max]="maxDate" [matDatepicker]="fromDatePicker" 
+        <mat-label>{{ 'labels.inputs.From Date' | translate }}</mat-label>
+        <input matInput [min]="minDate" [max]="maxDate" [matDatepicker]="fromDatePicker"
           required formControlName="loanFromDate" required>
```

HTMLHint (1.5.0)

[error] Duplicate of attribute name [ required ] was found.

(attr-no-duplication)